proxies/nodejs/app.ts

```diff
@@ -4,17 +4,18 @@ import Router from 'koa-router'
 import bodyParser from 'koa-bodyparser'
 import { handleUser } from './handlers/user'
 import { handleClient } from './handlers/client'
+import { handleLocation } from './handlers/location'
 
 type Data = {
-  clients: { [key: string]: DVCClient }
-  users: { [key: string]: DVCUser }
-  commandResults: { [key: string]: any }
-}
+  clients: { [key: string]: DVCClient };
```

Comment:
a bunch of objects still have semicolons after them, make sure to remove them all